### ISSUE TITLE
Getting Started with Gradient Boosted Decision Tree in Machine Learning

### ISSUE THREAD

**Issue by Muthami-John** [section-engineering-education/engineering-education#4556]
#### Pre-submission advice
By following all our pre-submission advice and reviewing our Resources folder, you will maximise your chances of your topic being approved.

We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner.

Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published.

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles that will help you propose and write a successful article.

**Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

#### Getting Started with Gradient Boosted Decision Tree in Machine Learning
Your title should be descriptive of the article/tutorial. Be Specific. 

Use keyword research to improve your article's SEO.

#### Proposed article introduction
Gradient-Boosted Decision Trees(GBDT) is a tree-based ensemble method that uses an ensemble of multiple decision trees to create a more powerful prediction model for classification and regression tasks. This algorithm builds a series of trees where each current tree is trained such that it attempts to correct the error of the previous tree in the series. Typically, the  GBDT uses a lot of shallow trees, usually known as *weak learners* in machine learning to make a model whose error gradually decrease as more trees are added. The GBDT algorithm is similar to AdaBoost, only that it has a depth of more than 1.

In this article, we shall talk about the intuition behind the GBDT algorithm, its pros and cons, and later learn how to implement it using the sklearn library in python. Also, we will learn how we can optimize it such that it doesn't over-learner the training phase and perform poorly on the testing phase. We make this possible by tunning its two crucial parameters, i.e., the learning rate and the max depth.
#### Key takeaways 
By the end of the article, the reader will know:
- What's the GBDT algorithm
- What's the difference between the GBDT and other ensemble techniques such as random forest, AdaBoost, and XGBoost
- How to implement this algorithm in python
- How to optimize this algorithm

#### Article quality
How is your article unique? Tell us what makes your approach different from similar articles that have published on the same topic?
Is yours more in-depth? Does it cover additional topics? Do you provide handy tips or anecdotal advice?

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)
